Doras/src/app/roles/roles.component.html

```diff
@@ -0,0 +1,16 @@
+<div class='container-fluid' data-interval="false">
+    <div class='col-12'>
+        <ngb-carousel data-interval="false">
+            <ng-template ngbSlide>
+                <app-table1></app-table1>
```

Comment:
come up with a better selector name! `app-table1` is not good naming convention!